### DISCUSSION TITLE
editOnClick error

### DISCUSSION THREAD

**Discussion by musumbidenis** [Power-Components/livewire-powergrid#1042] [Q&A]
I'm getting the following error while implementing editOnClick functionality:
_App\Http\Livewire\Products::onUpdatedEditable(): Argument #1 ($id) must be of type string, null given, called in C:\wamp64\www\rkprinters\vendor\power-components\livewire-powergrid\src\Traits\Listeners.php on line 22_

Someone kindly assist.

**Comment by shakeel2717** [comment]
you are using latest version?
